cloudflare.php

```diff
@@ -14,6 +14,15 @@
     exit;
 }
 
+const CF_MIN_PHP_VERSION = '5.3.10';
+const CF_MIN_WP_VERSION = '3.4';
```

Comment:
Its changed to 5.3.10 because the root cause of this bug is a user running 5.3.3, and the issue wasn't fixed until 5.3.9